usaspending_api/reporting/v2/views/agencies/agency_code/overview.py

```diff
@@ -77,12 +79,21 @@ def get_agency_overview(self):
                     .values("count"),
                     output_field=IntegerField(),
                 ),
+                gtas_total_budgetary_resources=Subquery(  # question for reviewer: since this is a dividend, are there circumstances that it would be 0 or nonexistent?
```

Comment:
For production data: there should not be an acceptable reason for any missing GTAS rows for all valid fiscal-year/-period combinations. This is because GTAS data are a dependency for DABS publication.

Reply:
thank you, removed comment